pyrobosim/pyrobosim/navigation/occupancy_grid.py

```diff
@@ -97,6 +97,10 @@ def save_to_file(self, folder, filename="world_map"):
         with open(yaml_file, "w") as f:
             yaml.dump(yaml_dict, f, sort_keys=False, default_flow_style=None)
 
+    def xy_from_world(self, x, y):
+        # Convert to grid coordinates
+        return x, y
```

Comment:
a) Does this function do anything right now?
b) Given my comment on maybe just having a `check_occupancy()` method (or maybe even two; one that uses grid coordinates and one that uses world), does this function need to even be here long term?
c) If this is going to stay, make sure to include docstrings

Reply:
> Does this function do anything right now?

Does not do anything right now, was going to implement this next
However considering the current comments,  I am a cleaner way would be to implicitly convert from world to grid coordinates in the `is_occupied()` method.